### DISCUSSION TITLE
Blur effect like Windows 10

### DISCUSSION THREAD

**Discussion by AoHeaven** [TomSchimansky/CustomTkinter#1054] [Q&A] [answered]
Can I make this on Frame widget?
Settings on Windows 10
![image](https://user-images.githubusercontent.com/95772447/212555692-454ad03c-8a0b-4379-873a-d9deb8f08625.png)
My app
![image](https://user-images.githubusercontent.com/95772447/212555728-dd3f8e8a-4102-4b9e-9861-0377e20fd4f0.png)

**Comment by Akascape** [comment, marked as answer]
@AoHeaven It can be achieved with the alpha attribute: `root.attributes("-alpha", value_closer_to_one)`
### Here is an example:
![Screenshot 2023-01-16 005634](https://user-images.githubusercontent.com/89206401/212562742-0f1693d1-42ae-45c3-9b77-7e6d9c39af23.jpg)
(I have tried to implement the dark mica theme)
```python
import customtkinter
from ctypes import windll, byref, sizeof, c_int

customtkinter.set_appearance_mode("Dark")

root = customtkinter.CTk()
root.configure(fg_color="#1e2434") # match the root window with mica theme color
root.attributes("-alpha", 0.992) # Add a slight alpha/transparency

# Change the header color with mica theme, (only for windows 11)
HWND = windll.user32.GetParent(root.winfo_id())
windll.dwmapi.DwmSetWindowAttribute(HWND, 1029, byref(c_int( 0x01)), sizeof(c_int))

root.mainloop()
```

**Reply by AoHeaven** [reply]
Thanks 😊
